MicroProfile-Rest-Client/tck-runner/pom.xml

```diff
@@ -136,14 +136,15 @@
         </profile>
         <!-- Payara 5 versions from 5.192 -->
         <profile>
-            <id>payara5.193+</id>
+            <id>payara5.192+</id>
             <activation>
                 <property>
                     <name>payara.version</name>
                     <value>/^[5-9]\.(19[2-4]|[2-9]\d{2})(?:-SNAPSHOT|\.\d*)?/</value>
```

Comment:
This regex isn't correct, it still matches against 5.194.

Reply:
Surely we only want it to be greater than 5.192 up to the next specified profile (your new one)?
Otherwise you're leaving it up to maven for which order the profiles get activated on, and therefore which version of Jersey is used.